### ISSUE TITLE
Restic environment variables break the app

### ISSUE THREAD

**Issue by arteymix** [flathub/org.gnome.DejaDup#33]
I use restic from the command line for my backups and export a few environment variables, notably `RESTIC_COMMAND_PASSWORD`. It breaks the Flatpak which cannot access the command.

This might be a more general thing with DejaDup however, the subprocess should have its environment stripped for the restic backend to prevent conflicting options.

Current workaround:

```bash
flatpak override --user org.gnome.DejaDup --unset-env RESTIC_PASSWORD_COMMAND
```

**Comment by mikix** [comment]
Good suggestion - implemented in main, will be in 46.0: https://gitlab.gnome.org/World/deja-dup/-/commit/0979705757d194791381dde6adec2d0647d4f72b

**Comment by arteymix** [comment]
Definitely the right thing to do!
